transcendence-app/src/chat/chat.controller.ts

```diff
@@ -408,21 +407,21 @@ export class ChatController {
     @Param('chatroomId', ParseUUIDPipe) chatroomId: string,
     @UploadedFile()
     file: Express.Multer.File,
-  ): Promise<AvatarResponseDto> {
+  ): Promise<Chatroom> {
     if (!file) {
       throw new UnprocessableEntityException();
     }
 
-    const avatar = await this.chatService.addAvatar(chatroomId, user, {
+    const updatedChatroom = await this.chatService.addAvatar(chatroomId, user, {
       filename: file.filename,
       path: file.path,
       mimetype: file.mimetype,
       size: file.size,
     });
 
-    if (!avatar) {
+    if (!updatedChatroom) {
       throw new ServiceUnavailableException();
     }
-    return avatar;
+    return updatedChatroom;
```

Comment:
There was an error when serializing the `AvatarResponseDto`, and we don't use the file from this response in the front end, so I changed the endpoints to return an updated user.
Please regenerate the webapp OpenAPI files.